### PR TITLE
Merge v0.2 to master

### PR DESCRIPTION
**Pull request by avclarke**

### CODE REVIEW

File: packages/launcher/src/renderer/launcher/PermissionsView.js [hunk 1]
```diff
@@ -224,21 +223,21 @@ export default class PermissionsView extends Component<Props, State> {
...
       if (Array.isArray(value)) {
         if (!value.length) {
           return null
         }
-        options = (
+        options = null
+        domainOptions = (
           <Domains key={key}>
-            {value.map(domain => (
+            {['google.com', ...value].map(domain => (
```
Comment (PaulLeCam) [inline packages/launcher/src/renderer/launcher/PermissionsView.js]: Any reason we have `google.com` hardcoded here please?

Comment (avclarke) [reply packages/launcher/src/renderer/launcher/PermissionsView.js]: Good spot, not sure why this was added, was it intentional to commit this @diogoperillo?

Comment (diogoperillo) [reply packages/launcher/src/renderer/launcher/PermissionsView.js]: Oops... sorry.. I left it there... My bad.

Comment (diogoperillo) [reply packages/launcher/src/renderer/launcher/PermissionsView.js]: here's the PR to fix it.
https://github.com/MainframeHQ/mainframe-os/pull/357

**Pull request merged**